### PR TITLE
Optionally pipe application container logs to master process log

### PR DESCRIPTION
**Pull request by chtyim**
- Supported different log level.

### CODE REVIEW

**Comment by sreevatsanraman** [comment]
LGTM :+1:

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -869,9 +869,10 @@
 
   <property>
     <name>master.collect.app.containers.log</name>
```
Comment (albertshau) [inline cdap-common/src/main/resources/cdap-default.xml]: loglevel instead of log would be a more descriptive name

**Comment by albertshau** [comment]
one minor comment about setting name, otherwise lgtm

File: cdap-common/src/main/resources/cdap-default.xml (hunk 1 above)
Comment (sreevatsanraman) [inline cdap-common/src/main/resources/cdap-default.xml]: Fixe.d

**Comment by sreevatsanraman** [comment]
Build: http://builds.cask.co/browse/CDAP-RBT612 

Will merge once the build passes.

File: cdap-common/src/main/resources/cdap-default.xml (hunk 1 above)
Comment (chtyim) [inline cdap-common/src/main/resources/cdap-default.xml]: Oops, forget to update that.

**Comment by sreevatsanraman** [comment]
Build passed: http://builds.cask.co/browse/CDAP-RBT612-1 merging.

**Pull request merged**